e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -80,4 +77,15 @@ export class TestWorkspaceUtil {
         }
     }
 
+    public async getIdOfRunningWorkspace(namespace: string): Promise<string> {
+        throw new Error("Method not implemented.");
```

Comment:
What approach do you mean? Throwing an error?

Reply:
I mean approach with the use of plugs instead of implementing realization

Reply:
I would say let's merge this PR, because I would prefer to not make this PR any bigger then it is and because we don't use/need these methods here right now and implement these methods in following PR (Katka will need this anyway for downstream work :-D).